docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -66,7 +66,7 @@ The core provided categories are:
 category: 'widgets',
 ```
 
-Plugins and Themes can also register [custom block categories](../docs/extensibility/extending-blocks/#managing-block-categories).
+Plugins and Themes can also register [custom block categories](/docs/extensibility/extending-blocks/#managing-block-categories).
```

Comment:
All link updated in this file seem to be outdated.

Reply:
yes, it's referring to the old structure.